core/station/impl/src/models/request_policy_rule.rs

```diff
@@ -44,6 +44,19 @@ pub enum RequestPolicyRule {
     NamedRule(NamedRuleId),
 }
 
+impl RequestPolicyRule {
+    pub fn has_named_rule_id(&self, named_rule_id: &NamedRuleId) -> bool {
+        match self {
+            RequestPolicyRule::NamedRule(id) => id == named_rule_id,
+            RequestPolicyRule::And(rules) | RequestPolicyRule::Or(rules) => rules
+                .iter()
+                .any(|rule| rule.has_named_rule_id(named_rule_id)),
+            RequestPolicyRule::Not(rule) => rule.has_named_rule_id(named_rule_id),
+            _ => false,
```

Comment:
Could we please expand the wildcard so that the compiler forces us to update this definition upon changes to `RequestPolicyRule`?